pkg/auth/auth_config_test.go

```diff
@@ -94,7 +85,7 @@ type ConfigTest struct {
 
 func (c *ConfigTest) Load() *auth.AuthConfig {
 	config, err := c.svc.LoadConfig()
-	c.AssertNoError(err)
+	require.NoError(c.t, err)
```

Comment:
OOI, what's the difference between `assert` and `require`?

Reply:
`require` stops immediately the tests without executing the follow up tests while `aseert` still keeps the test execution till to the end.